spec/rubocop/cop/lint/empty_conditional_body_spec.rb

```diff
@@ -347,10 +358,10 @@ class Foo
     it 'registers an offense for multi-line value omission in `unless`' do
       expect_offense(<<~RUBY)
         var =
+          # This is the value of `other:`, like so: `other: condition || other_condition`
           unless object.action value:, other:
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Avoid `unless` branches without a body.
-            condition || other_condition # This is the value of `other:`, like so:
-                                         # `other: condition || other_condition`
```

Comment:
NOTE: This is an intentional change to treat it as an `if` that does not contain comments.